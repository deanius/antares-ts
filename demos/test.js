const Demos = {
    writeFileDemo: require('./01-write-file')
}

let output = ''
const appendLine = (s) => {output = output + `${s}\n`}
const append = (s) => { output += s }

describe("writeFileDemo", () => {
    it("should write a file")
    it("should write to stdout", async () => {
        const subject = Demos.writeFileDemo
        const stdout = {write: jest.fn(appendLine)}
        await subject({ stdout })
        
        expect(output).toMatchSnapshot()
    })
})
