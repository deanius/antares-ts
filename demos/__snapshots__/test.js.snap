// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`writeFileDemo should write to stdout 1`] = `
"TODO write file
"
`;
